### ISSUE TITLE
RareCheese - C83224, low stock

### ISSUE THREAD

**Issue by kvalek** [Sorakage033/SlimeVR-CheeseCake#20]
JLCPCB has a low stock on the C83224 (M3406-ADJ). Is the C22462728 (MT3406) a suitable replacement?

**Comment by Sorakage033** [comment]
Yes of course, and you can also use TI's TLV62569DBVR (C141836) instead, they are pin-to-pin compatible.
